src/Sylius/Bundle/OrderBundle/Resources/config/doctrine/model/Adjustment.orm.xml

```diff
@@ -45,6 +45,10 @@
         <many-to-one field="orderItem" target-entity="Sylius\Component\Order\Model\OrderItemInterface" inversed-by="adjustments">
             <join-column name="order_item_id" referenced-column-name="id" nullable="true" on-delete="CASCADE"/>
         </many-to-one>
```

Comment:
sounds good to come back to it later.

Although I think you'll find it gets unnecessarily complicated to support both and if you can provide 'rolling up' of values at the display level it won't be needed. 

We'll see...